src/server/graphql/queries/organization.js

```diff
@@ -0,0 +1,39 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql';
+import getRethink from 'server/database/rethinkDriver';
+import Organization from 'server/graphql/types/Organization';
+import {getUserId} from 'server/utils/authorization';
+import {BILLING_LEADER} from 'universal/utils/constants';
+
+export default {
+  type: Organization,
+  args: {
+    orgId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'the orgId'
+    }
+  },
+  description: 'all the notifications for a single user',
+  resolve: async (source, {orgId}, {authToken}) => {
+    const r = getRethink();
+
+    // AUTH
+    const userId = getUserId(authToken);
+
+    const org = await r.table('Organization').get(orgId);
+
+    const {orgUsers} = org;
+    const myOrgUser = orgUsers.find((user) => user.id === userId);
+    if (!myOrgUser || myOrgUser.role !== BILLING_LEADER) {
+      throw new Error('Must be a billing leader');
```

Comment:
when we added organizations, we basically _only_ wanted it to serve 1 purpose: billing. we don't want folks to hesitate to create teams, because creating many, small teams is the secret to success.  but we gotta get paid, so organizations came to be (similar to the github structure now).

billing leader is required because just because you're on a team doesn't mean you should see everyone in the company. For example, if i invite a contractor (or a competitor) to my team, i don't want him to see everyone & steal them away from me. He should should only be able to see what i want him to see: the team.

Reply:
Thanks! Makes sense to me!

For consistency, it'd be cool to extract this into a function, similar to `requireSUOrTeamMember`.